debian/rules

```diff
@@ -85,6 +84,7 @@ override_dh_installinit:
 	install -m0644 systemd/ceph-create-keys@.service debian/ceph-base/lib/systemd/system
 	install -m0644 systemd/ceph-osd@.service debian/ceph-osd/lib/systemd/system
 	install -m0644 systemd/ceph-disk@.service debian/ceph-osd/lib/systemd/system
+	install -m0644 systemd/rbdmap.service debian/ceph/lib/systemd/system
```

Comment:
@ktdreyer maybe we should install `rbdmap.service` into `debian/ceph-common/lib/systemd/system` instead? see http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-deb-trusty-amd64-basic/log.cgi?log=d226051ec096b331497c3c90ff6767670302d3a9

```
install -m0644 systemd/rbdmap.service debian/ceph/lib/systemd/system
install: cannot create regular file ‘debian/ceph/lib/systemd/system’: No such file or directory
make[1]: *** [override_dh_installinit] Error 1
make[1]: Leaving directory `/srv/autobuild-ceph/gitbuilder.git/build/out~/ceph-11.0.0-1613-gd226051'
make: *** [binary] Error 2
error: dpkg-buildpackage: fakeroot debian/rules binary gave error exit status 2
```